lib/CMakeLists.txt

```diff
@@ -1,3 +1,5 @@
-add_library(disccord SHARED client.cpp)
+set(SOURCE_FILES client.cpp)
+
+add_library(disccord SHARED ${SOURCE_FILES})
```

Comment:
Assuming disccord is a shared lib. Change "SHARED" to "STATIC" if that ain't the case. You can also add `add_library(disccord_s STATIC ${SOURCE_FILES})` if you want both shared and static libs